code/datums/elements/honkspam.dm

```diff
@@ -0,0 +1,24 @@
+/**
+ * This used to be in paper.dm, it was some snowflake code that was
+ * used ONLY on april's fool. We moved it to an element so it could be used in other places.
```

Comment:
This doc-comment is completely and utterly useless. It doesn't say anything about what it actually *does*.

Comments should not say what code *used* to be--it's not useful. It should say what it is *now*.